frontend/components/MainPageV1/Home/AgentInto/AgentRunButton/AgentRunningButton.tsx

```diff
@@ -0,0 +1,45 @@
+import { Button } from 'antd';
+import { useCallback } from 'react';
+
+import { MiddlewareDeploymentStatusMap } from '@/constants/deployment';
+import { useElectronApi } from '@/hooks/useElectronApi';
+import { useService } from '@/hooks/useService';
+import { useServices } from '@/hooks/useServices';
+import { ServicesService } from '@/service/Services';
+
+export const AgentRunningButton = () => {
+  const { showNotification } = useElectronApi();
+
+  const {
+    selectedService,
+    isFetched: isLoaded,
+    overrideSelectedServiceStatus,
+  } = useServices();
+
+  const serviceConfigId =
+    isLoaded && selectedService?.service_config_id
+      ? selectedService.service_config_id
+      : '';
+  const { service } = useService(serviceConfigId);
+
+  const handlePause = useCallback(async () => {
+    if (!service) return;
+
```

Comment:
previously we had here commented out `setPaused(true);` I removed it completely and don't see where it's used. do you think it's still needed, or maybe we can remove it completely from the provider?

Reply:
realised it's used, but I removed from where it's not